src/pages/MessagesAndFriends.tsx

```diff
@@ -70,10 +77,24 @@ const MessagesAndFriends = () => {
     goToNextPotentialFriend(currentPotentialFriend)
   }
 
+  const isLiked = (accountId: string, likedUsersArray: string[]): boolean => {
+    const likedMe = likedUsersArray.includes(accountId)
+    if (likedMe) {
```

Comment:
I would recommend not to use such conditions, it's like red flag - means smth definitely wrong. 
`const isLiked = (accountId: string, likedUsersArray: string[]): boolean => {
          return likedUsersArray.includes(accountId)
  }`

Reply:
Thanks. Fixed it.